Makefile

```diff
@@ -21,3 +23,10 @@ refman-html:
 
 stdlib-html:
 	$(DUNE) build --root . @stdlib-html
+
+clean:
+	find . -name '*.vo' \
+		-o -name '*.vok' \
+		-o -name '*.vos' \
+		-o -name '*.glob' \
+		-exec rm -vf {} +
```

Comment:
seems incomplete
at least missing the .d and the coq-native files
you could call rocq makefile's make clean if the Makefile.coq exists
also you could rm -rf _build

Reply:
Thank you. Didn't know about these. Can you give me a pointer to these `.d` and `coq-native` files?